### ISSUE TITLE
Project is not an MMO

### ISSUE THREAD

**Issue by micklh** [mcMMO-Dev/mcMMO#2756]
To quote mcMMO itself:
> The goal of mcMMO is to take core Minecraft game mechanics and expand them into add an extensive and quality RPG experience.

...or...

> mcMMO's main function is to incorporate an RPG-like skill system that players will have access to right from the beginning.

I see mention to quality RPG elements, and only quality RPG elements.

Now please consider the acronym "MMORPG" (Massively-Multiplayer Online Role Playing Game)
While MMO does *usually* imply MMORPG, it also **directly** conveys massively multiplayer online.

mcMMO does not provide any MMO features beyond vanilla Minecraft. Even assuming MMO is short for MMORPG does not rectify the situation, as it plainly is **not** an "MMORPG". It is simply an "RPG".
In conclusion, either the project name containing "MMO" is a logical error, or the project's lack of any type of MMO feature whatsoever is a logical error.

So, are you looking to gain MMO functionality, or is the project name simply misleading / malicious?

**Comment by DogsPaw** [comment]
Leave.

**Comment by t00thpick1** [comment]
I have seen servers running mcMMO with well into the Hundreds of players online.  To compare with some MMO's this is a valid number.  Therefore the name is valid.  You could argue that any small server is "misusing" the plugin, but honestly you are just trying to be a special snowflake for the sake of an argument.

**Comment by micklh** [comment]
No need to become defensive, there is no criticism here. Nobody wishes to argue with you over GitHub.

There is simply an objective logical error: As vanilla Minecraft provides a server which by your definition is already "MMO", the name "mcMMO" is not descriptive because Minecraft provides the "mc" and "MMO" elements while the actual RPG functionality is not represented. I am sorry for suggesting an easy way to objectively improve your open source project. I will refrain from pointing out more of the numerous typographical errors littered throughout the project's literature, and I did not intend to embarrass anyone with the very first quote.


> but honestly you are just trying to be a special snowflake for the sake of an argument.

It appears that you are not only projecting your own personal behaviors, but are also deeply confident in your assumptions. So I also am sorry if this is offensive to any clinical narcissists, but this project is not some kind of "special snowflake" that attracts people just wishing to argue about trivial nothingness.

Happy hacking!
